drake_ros/drake_ros/systems/movable_point.py

```diff
@@ -0,0 +1,117 @@
+import geometry_msgs.msg
+from pydrake.systems.framework import BasicVector_, LeafSystem
+from visualization_msgs.msg import (InteractiveMarker,
+                                    InteractiveMarkerControl,
+                                    InteractiveMarkerFeedback, Marker)
+
+# from interactive_markers import InteractiveMarkerServer
+
+# from tf2_ros import Buffer
+# from tf3_ros import TransformListener
+
+
+
+
+class MovablePoint(LeafSystem):
```

Comment:
I'm guessing this system is not needed anymore since we're not doing IK?

Reply:
Yup. Removed unused `MovablePoint` system in 3860461